example/signed-request.sh

```diff
@@ -1,30 +1,103 @@
 #!/usr/bin/env bash
-# Make a signed request to an OpenToken API.  This is a proof of concept
-# illustrating that signed requests can be made using only the Bash shell.
+#/ Make a signed request to an OpenToken API.  This is a proof of concept
+#/ illustrating that signed requests can be made using only the Bash shell.
+#/
+#/ Requires the following environment variables:
+#/
+#/ * OPENTOKEN_API: URL typically used for self discovery.
+#/   eg. http://api.example.com/
+#/ * OPENTOKEN_CODE: The API code.
+#/ * OPENTOKEN_SECRET: The API secret.
+#/ * DEBUG: When set to a non-empty value, this turns on some debugging.
```

Comment:
Updated comment:

    #/ * DEBUG: When set to a non-empty value, this turns on debugging output,
    #/   including line-by-line output and showing the entire curl command.